src/components/Error/Error.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
```

Comment:
```suggestion
import { useEffect, FC } from 'react';
```

Reply:
The error is thrown because FC writes that they are not used